src/dview/dvplotctrl.cpp

```diff
@@ -449,7 +449,7 @@ void wxDVPlotCtrl::SelectDataIndex(size_t index, bool allTabs)
 {
 	//Only select this data on the slower tabs (which require sorting) if allTabs is true.
 
-	if (index < 0 || index >= m_dataSets.size())

```

Comment:
This is probably warning because a size_t cannot logically be < 0.

Reply:
That is exactly why it is warning.